cmd/backup-manager/app/util/util.go

```diff
@@ -389,13 +389,23 @@ func GetCommitTsFromBRMetaData(provider v1alpha1.StorageProvider) (uint64, error
 }
 
 // ConstructArgs constructs the rclone args
-func ConstructArgs(conf string, opts []string, command, source, dest string) []string {
+func ConstructArgs(conf string, opts []string, command, source, dest string, rcloneLog bool) []string {
 	var args []string
+	defaultLog := true
 	if conf != "" {
 		args = append(args, conf)
 	}
 	if len(opts) > 0 {
-		args = append(args, opts...)
+		for _, opt := range opts {
+			if rcloneLog == false && strings.HasPrefix(opt, "-v") {
```

Comment:
Also have to consider `--verbose`.

Reply:
Could you please add UT for this function?

Reply:
UT has added, fix `--verbose`.